src/txmempool.cpp

```diff
@@ -395,7 +395,7 @@ bool CTxMemPool::addUnchecked(const uint256& hash, const CTxMemPoolEntry &entry,
 {
     NotifyEntryAdded(entry.GetSharedTx());
     // Add to memory pool without checking anything.
-    // Used by main.cpp AcceptToMemoryPool(), which DOES do
+    // Used by validation.cpp AcceptToMemoryPool(), which DOES do
```

Comment:
grep, ack, tag files, ... there are many ways to find where the AcceptToMemoryPool() definition is. Naming the source file doesn't add any clarity here, and as you say adds cost to future refactors.